plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -136,14 +137,18 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
 
     new ServerLaunchMonitor(launch, server).engage();
 
+    String vmArgumentString = getVMArguments(configuration);
+    // This string is exactly as supplied by the user in the dialog box
+    
+    List<String> vmArguments = Arrays.asList(vmArgumentString.split("\\s+"));
```

Comment:
Good point. The issue is quoted arguments, right? 

E.g. --foo="some value with spaces"